packages/node/src/policy-util.js

```diff
@@ -61,92 +137,99 @@ export const readPolicyOverride = async (
+ * @param {LoadPoliciesOptions} [options] Options
  * @returns {Promise<LavaMoatPolicy>}
- * @throws If a policy is invalid _and/or_ if policy overrides were provided
- *   _and_ are invalid
+ * @throws If a policy is invalid
  * @public
  */
 export const loadPolicies = async (
-  policy = constants.DEFAULT_POLICY_PATH,
-  policyOverride = constants.DEFAULT_POLICY_OVERRIDE_PATH
+  policyOrPolicyPath,
+  {
+    readFile = nodeFs.promises.readFile,
+    projectRoot = process.cwd(),
+    ...options
+  } = {}
 ) => {
+  policyOrPolicyPath ??= nodePath.resolve(
+    projectRoot,
+    constants.DEFAULT_POLICY_PATH
+  )
+
+  /** @type {LavaMoatPolicy | string | URL} */
+  let policyOverrideOrPolicyOverridePath
+  if (hasValue(options, 'policyOverride')) {
+    policyOverrideOrPolicyOverridePath = options.policyOverride
+  } else if (hasValue(options, 'policyOverridePath')) {
+    policyOverrideOrPolicyOverridePath = options.policyOverridePath
+  } else {
+    // _if_ we have a policy path, we should compute the policy override path as its sibling.
+    // if not, then we just have to guess.
+    if (isString(policyOrPolicyPath) || policyOrPolicyPath instanceof URL) {
+      const policyPath = toPath(policyOrPolicyPath)
+      policyOverrideOrPolicyOverridePath = nodePath.join(
+        nodePath.dirname(policyPath),
+        constants.DEFAULT_POLICY_OVERRIDE_FILENAME
+      )
+    } else {
+      policyOverrideOrPolicyOverridePath = nodePath.resolve(
+        projectRoot,
+        constants.DEFAULT_POLICY_OVERRIDE_PATH
+      )
+    }
+  }
+
   /**
-   * @type {[
-   *   Promise<LavaMoatPolicy>,
-   *   Promise<LavaMoatPolicyOverrides | undefined>,
-   * ]}
+   * @type {[Promise<LavaMoatPolicy>, Promise<LavaMoatPolicy | undefined>]}
    */
   const promises = /** @type {any} */ ([])
-  if (isObject(policy)) {
+  if (isPathLike(policyOrPolicyPath)) {
+    promises.push(readPolicy(policyOrPolicyPath, { readFile }))
+  } else {
     promises.push(
       Promise.resolve().then(() => {
-        assertPolicy(policy)
-        return policy
+        assertPolicy(policyOrPolicyPath)
+        return policyOrPolicyPath
       })
     )
-  } else {
-    promises.push(readPolicy(policy))
   }
-  if (isObject(policyOverride)) {
+  if (isPathLike(policyOverrideOrPolicyOverridePath)) {
     promises.push(
-      Promise.resolve().then(() => {
-        assertPolicyOverride(policyOverride)
-        return policyOverride
+      maybeReadPolicyOverride(policyOverrideOrPolicyOverridePath, {
+        readFile,
       })
     )
   } else {
-    promises.push(readPolicyOverride(policyOverride))
+    promises.push(
+      Promise.resolve().then(() => {
+        try {
+          assertPolicy(policyOverrideOrPolicyOverridePath)
+          return policyOverrideOrPolicyOverridePath
+        } catch (err) {
+          throw new Error(
+            `Invalid LavaMoat policy overrides; does not match expected schema`,
+            { cause: err }
+          )
+        }
```

Comment:
(nit) this bit repeats a bunch, could go into a function. the only difference is the word "overrides" on the message.

Reply:
yes, but felt like premature abstraction to me, so I didn't. might as well

Reply:
- [ ] make a function

Reply:
nevermind, too difficult to type correctly.